base.py

```diff
@@ -498,10 +498,21 @@ def update_time(scene):
 class MBDynReadLog(bpy.types.Operator):
     """ Imports MBDyn nodes and elements by parsing the .log file """
     bl_idname = "animate.read_mbdyn_log_file"
-    bl_label = "MBDyn .log file parsing"
```

Comment:
If I may, I don't see a problem w.r.t the user, as the user reads that label only when the dialog box pops up. I agree it looks confusing to a developer to read, but maybe we could document it properly?